src/code/bank0.asm

```diff
@@ -6604,24 +6604,24 @@ label_3500::
     cp   $09                                      ; $3503: $FE $09
     jr   nz, label_350E                           ; $3505: $20 $07
     ldh  a, [hMapRoom]                            ; $3507: $F0 $F6
-    cp   $97            ; Room with the giant bombable skull thing ; $3509: $FE $97
+    cp   INDOOR_GIANT_SKULL_ROOM                  ; $3509: $FE $97
     ret  nz                                       ; $350B: $C0
     jr   label_3527                               ; $350C: $18 $19
 
 label_350E::
     cp   $E1                                      ; $350E: $FE $E1
     jr   nz, label_351D                           ; $3510: $20 $0B
     ldh  a, [hMapRoom]                            ; $3512: $F0 $F6
-    cp   $0E            ; Eagle's Tower room      ; $3514: $FE $0E
+    cp   OVERWORLD_EAGLE_TOWER_ENTRANCE           ; $3514: $FE $0E
     ret  z                                        ; $3516: $C8
-    cp   $0C            ; Tal-Tal Heights, 1N1E of the drainable water above Angler's Tunnel ; $3517: $FE $0C
+    cp   UNKNOWN_ROOM_0C                          ; $3517: $FE $0C
     ret  z                                        ; $3519: $C8
-    cp   $1B            ; The drainable water above Angler's Tunnel ; $351A: $FE $1B
+    cp   UNKNOWN_ROOM_1B                          ; $351A: $FE $1B
     ret  z                                        ; $351C: $C8
 
 label_351D::
     ldh  a, [hMapRoom]                            ; $351D: $F0 $F6
-    cp   $80            ; Mysterious Forest enterance from Mabe (?) ; $351F: $FE $80
+    cp   IS_INDOOR                                ; $351F: $FE $80
```

Comment:
Map rooms go from $00 to $FF on all maps, overworld and indoor. Not sure if this `IS_INDOOR` is correct then. It's more probably identifying the first part of the overworld rooms?